.travis.yml

```diff
@@ -26,11 +26,14 @@ matrix:
         apt:
           sources:
             - sourceline: 'ppa:ubuntu-toolchain-r/test'
+            - sourceline: 'deb https://apt.kitware.com/ubuntu/ xenial main'
+              key_url: 'https://apt.kitware.com/keys/kitware-archive-latest.asc'
+            - sourceline: 'ppa:mhier/libboost-latest'
           packages:
             - gcc-7
             - g++-7
             - cmake
-            - libboost-all-dev
+            - libboost1.70-dev
```

Comment:
Just to clarify, is this to confirm that we're good with the latest boost or is this because we're now requiring a more recent version? Are there additional warnings we need to suppress with older boost versions?

Reply:
This is actually due to an interaction between:
- older versions of boost
- older GCC versions
- -Wimplicit-fallthrough
- our vendored libllvm

Basically, I was using `BOOST_FALLTHROUGH` to silence the implicit fallthrough warnings in libllvm. That doesn't work under certain older configurations.

We can relax the libboost constraint once we remove libllvm for instance.

Reply:
Got it. Relaxing this constraint after removing the vendored LLVM code is a great plan. Thanks for figuring that out.